### PR TITLE
Calypsoify: Update default colors to match WordPress.com

### PR DESCRIPTION
**Pull request by blowery**
#### Changes proposed in this Pull Request:

* Update the colors used by Calypsoify to match the new default colors on WordPress.com

#### Testing instructions:

* Open the plugins screen with `?calypsoify=1`. You should see the calypsoified plugins screen
* Verify that the colors look the same as those used on WordPress.com with the default theme picked

#### Proposed changelog entry for your changes:

* Calypsoify: Update colors and color usage to match recent changes on WordPress.com

### CODE REVIEW

**Comment by blowery** [comment]
This is a split off from #10992. It extracts the color moves and applies them as defaults instead of hiding them under a flag. WIP.

File: modules/calypsoify/_colors.scss [hunk 1]
```diff
@@ -1,70 +1,70 @@
+@import 'muriel-wpcom';
 // Colors that aren't pulled from Calypso or Core:
 
-$_0099d8: #0099d8; // Pacific Blue
-$_0096c3: #0096c3; // Pacific Blue
-$_008ab3: #008ab3; // Bondi Blue
-$_004967: #004967; // Prussian Blue
+$_0099d8: hsl(198, 100%, 42%); // Pacific Blue
```
Comment (blowery) [inline modules/calypsoify/_colors.scss]: oops. i think i switched these to hsl while testing... should move back to hex or replace them entirely with more semantic names

**Comment by timmyc** [comment]
@blowery so will this only support the default scheme, or if someone toggles to "Classic Blue" will it be picked up and displayed here? I'm not seeing where that might be toggled in the code - sorry if I've just missed it though.

Also was chatting with some folks on the Hydra team, and they have a desire to manipulate their css in the WooCommerce Services plugin if it is being used in a calypsoified view. What do you think about adding some body classes in this PR too - one that denotes the page is being rendered in calypsoify ( maybe `.is-calypsoify` ) and another to match the color scheme body class logic that is being used in Calypso/wpcom?

**Comment by blowery** [comment]
> so will this only support the default scheme, or if someone toggles to "Classic Blue" will it be picked up and displayed here? I'm not seeing where that might be toggled in the code - sorry if I've just missed it though.

@timmyc default theme only for this PR, we hope to add theming support in a future PR.

> What do you think about adding some body classes in this PR too?

Sure? Sounds pretty speculative as is, so it might be best to flesh out the requirements a bit more.

**Comment by timmyc** [comment]
> Sure? Sounds pretty speculative as is, so it might be best to flesh out the requirements a bit more.

I'll open up a stand-alone PR to speak to it a bit further, and provide some use cases where this would be helpful.

**Comment by jeherve** [comment]
@blowery Is this ready for review, or can it wait until the next release, in March?

**Comment by jeherve** [comment]
It seems to look good on a first pass. I've tagged our designers for a review.

---- 

I spot a few places in the block editor where we could use our own colors, for example `.components-form-toggle.is-checked::before`, `.editor-block-list__block input[type=text]:focus`, `.wp-core-ui .button-primary`, `.wp-block-jetpack-tiled-gallery .tiled-gallery__item.is-selected`. Those can be found when playing around with the different Jetpack blocks. It's not necessarily related to your PR though, as most often those are colors that have not been changed by Calypsoify at all.

**Comment by rickybanister** [comment]
This looks pretty close. The plugins main content area looks good, but I noticed the notification dot in the masterbar looks off (I'm not sure if that's within the scope of this PR or not...)

image

**Comment by blowery** [comment]
@jeherve ideally we'd like to get it in asap. @flootr any obvious things left to do here?

**Comment by blowery** [comment]
Looks like we missed the code freeze though. I guess we wait. :)

**Comment by jeherve** [comment]
> Looks like we missed the code freeze though. I guess we wait. :)

If we get @rickybanister's comment figured out, I can see if we can ship these changes in a Beta 2 tomorrow

**Comment by flootr** [comment]
> If we get @rickybanister's comment figured out, I can see if we can ship these changes in a Beta 2 tomorrow

This is an interesting one. Unfortunately, it's using `!important` (😞) to apply color for the notification bell. Additionally it's also overriden by `admin-bar-v2.css` which is loaded _after_ our calypsoify css.

screenshot 2019-01-31 at 11 56 10

I'm not really sure what to do at this point. Can we switch order of loading those styles? Would that break other stylings?

**Comment by jeherve** [comment]
When I switched from viewing all plugins to "Add New", there is a flash with the plugin icon:

http://cl.ly/35957ed118be

Is that icon supposed to be black, white, or blue?

**Review by jeherve — COMMENTED** [review]
Other than the comment I left [here](https://github.com/Automattic/jetpack/pull/11115#issuecomment-459396349), this looks good to me 👍

**Comment by blowery** [comment]
> When I switched from viewing all plugins to "Add New", there is a flash with the plugin icon

how weird. feels like a flash of unstyled content in a way..

> Is that icon supposed to be black, white, or blue?

@drw158 have an opinion here?

**Comment by drw158** [comment]
Looks like maybe a rogue `active` style somewhere. Here are the styles for that icon:

default
screen shot 2019-01-31 at 11 57 33 am

hover
screen shot 2019-01-31 at 11 57 46 am

selected
screen shot 2019-01-31 at 11 57 40 am

**Comment by flootr** [comment]
That blue metioned in https://github.com/Automattic/jetpack/pull/11115#issuecomment-459443502 is probably not possible due to the way how these icons are added - see p1548958258494200-slack-CDZD4K2CD for more details.

The short version this icon is added via `style` attribute on the element as background image which make it hard to style it properly. I'm looking into what's possible.

**Comment by flootr** [comment]
By the way I was able to replicate this bug in the current version of Calypsoify as well:

brightness-svg

**Comment by blowery** [comment]
OK, thanks for the investigation @flootr. I think we're good to land then?

**Comment by flootr** [comment]
> I think we're good to land then?

Should be good to go 👍

**Review by jeherve — APPROVED** [review]
Looking good now. Merging!

**Pull request merged**